torchvision/models/video/swin_transformer.py

```diff
@@ -124,6 +124,7 @@ def shifted_window_attention_3d(
     dropout: float = 0.0,
     qkv_bias: Optional[Tensor] = None,
     proj_bias: Optional[Tensor] = None,
+    training: bool = False,
```

Comment:
Should we set the default to True since it's the default of `torch.nn.functional.dropout`, and also the default of `shifted_window_attention` ?

Reply:
Well caught this was mistake it should have been True